src/main/java/uk/gov/justice/digital/config/JobArguments.java

```diff
@@ -330,9 +332,25 @@ public ImmutableSet<String> getBucketsToDeleteFilesFrom() {
                 .map(String::trim)
                 .filter(item -> !item.isEmpty())
                 .collect(Collectors.toSet());
+        if (buckets.isEmpty()) throw new IllegalStateException("Argument " + ALLOWED_S3_FILE_EXTENSIONS + " evaluated to empty set");
```

Comment:
Should ALLOWED_S3_FILE_EXTENSIONS be FILE_DELETION_BUCKETS in this message?

Reply:
Good catch. Should be fixed now. Thanks 👍